### PR TITLE
Add support for Nodejs 10

### PR DESCRIPTION
**Pull request by Kocal**
I followed what it has been done in #68, I hope that will works :sweat_smile:

### CODE REVIEW

**Comment by Kocal** [comment]
Oh, I didn't see the pattern.
It's fixed now. :+1:

File: manala.apt/vars/main.yml [hunk 1]
```diff
@@ -145,6 +145,9 @@ manala_apt_repositories_patterns:
   nodesource_9:
     source: deb https://deb.nodesource.com/node_9.x {{ ansible_distribution_release }} main
     key: nodesource
+  nodesource_10:
+    source: deb https://deb.nodesource.com/setup_10.x {{ ansible_distribution_release }} main
```
Comment (chalasr) [inline manala.apt/vars/main.yml]: should be `/node_10.x`, right?

Comment (Kocal) [reply manala.apt/vars/main.yml]: Nice catch, but in the [documentation](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) they use `/setup_10.x`. :thinking: 

I will change for `/node_10.x`

**Comment by nervo** [comment]
Wow, thx a lot for your work, dude !

**Pull request merged**